test/functional/local_transaction_controller_test.rb

```diff
@@ -186,56 +180,38 @@ def subscribe_logstasher_to_postcode_error_notification
       setup do
         stub_mapit_does_not_have_a_bad_postcode("BLAH")
 
-        subscribe_logstasher_to_postcode_error_notification
-
         post :search, params: { slug: "send-a-bear-to-your-local-council", postcode: "BLAH" }
       end
 
-      should "expose the 'invalid postcode format' error to the view" do
-        location_error = assigns(:location_error)
-        assert_equal location_error.postcode_error, "invalidPostcodeFormat"
-      end
-
-      should "log the 'invalid postcode format' error to the view" do
-        assert_equal(LogStasher.store["postcode_error_notification"], postcode_error: "invalidPostcodeFormat")
+      should "responsds successfully with an error message" do
```

Comment:
One last little thing which is that there a typo in 'responsds`a couple of times in this file @kevindew

Reply:
Oops - thanks for catching. And to think I c+p'd that 3 times.

Reply:
hehe 👍